### DISCUSSION TITLE
Integrating external MongoDB server to CVAT server

### DISCUSSION THREAD

**Discussion by 40016044** [cvat-ai/cvat#4035] [Q&A]
We are using CVAT with MongoDB rather than PostgreSQL. We have created MongoDB server and CVAT server VMs in the OpenStack cloud.  While running the migration from the Django application to the MongoDB database by the below command we are getting errors like mentioned below.
# python manage.py makemigrations

ERROR:

pymongo.errors.ServerSelectionTimeoutError: 10.40.4.8:27017:[Errno 111] Connection refused, Timeout: 30s, Topology Description: <TopologyDescription id:61badd0039e87cf84e85a55b, topology_type:Single, servers: [<ServerDescription('10.40.4.8',27017) server_type: Unknown, rtt:None, error=AutoReconnect('10.40.4.8:27017:[Errno 111] Connection refused')>]>

**Comment by azhavoro** [comment]
Unfortunately, we only support Postgres as production db. Most likely CVAT will work with MySQL-like databases as well.
P.S. what are the benefits of using mongodb instead of postgres?
